### PR TITLE
Helm logo update

### PR DESCRIPTION
**Pull request by flynnduism**
This updates the site header to remove the Microsoft logo at the top of the site - to edit the branding to reflect the community effort.

Updated top logo:

screen shot 2017-11-14 at 4 21 52 pm

Contributor logos:

screen shot 2017-11-14 at 4 22 00 pm

**I haven't updated this yet** - wondering who else should be included? Ideally we would give shoutouts to all contributors equally. _Seeking input!_

### CODE REVIEW

**Comment by bacongobbler** [comment]
We should at the very least add the logos from the core maintainers' companies (with their permission of course! 😄)

From their respective GitHub profiles:

- Nike (@thomastaylor312)
- Intel (@jascott1)
- Mirantis (@nebril)
- IBM (@seh)

And for chart maintainers:

- Ticketmaster (@mgoodness)
- NBC Universal (@scottrigby)
- Samsung SDS (@mattfarina)
- codecentric AG (@unguiculus)

If I missed anyone let me know!

**Comment by scottrigby** [comment]
@flynnduism nice work!

@bacongobbler sounds good to me! But don't use the logo on the [NBCUTechnology](https://github.com/NBCUTechnology) github org, as that's the wrong logo (ps that repo is non-representative of the org's public projects, but that's another story). This is the correct logo: https://en.wikipedia.org/wiki/NBCUniversal#/media/File:NBCUNI_Logo.svg

File: index.html [hunk 1]
```diff
@@ -167,10 +167,32 @@ <h2 class="text-center">Join the Community</h2>
...
+
+            <hr>
+            <p class="contributors">
+              Helm is supported by and built with a community of over 250 <a href="https://github.com/kubernetes/helm/blob/master/OWNERS">contributors</a>.
+            </p>
+            <hr>
+
+            <div class="row helm-contrib-logos contrib-logos-1">
+              <a target="_blank" href="http://kubernetes.io/"><img src="{{site.baseurl}}/assets/images/logowall/microsoft.png" alt="Microsoft"></a>
+              <a target="_blank" href="http://kubernetes.io/"><img src="{{site.baseurl}}/assets/images/logowall/google.png" alt="Microsoft"></a>
```
Comment (bacongobbler) [inline index.html]: looks like the alt tags need updating; they all say "Microsoft".

**Comment by flynnduism** [comment]
Thanks @bacongobbler, updated!

**Comment by flynnduism** [comment]
Screenshot:

screen shot 2017-11-15 at 4 04 36 pm

**Comment by thomastaylor312** [comment]
@flynnduism Let's pull out Nike. I work for them, but I don't work on Helm as a Nike employee. Also, I think there is legal stuff involved, which is always...fun. I'll check in to what options we have though for the future. Thanks for putting this together!

**Comment by bacongobbler** [comment]
We should just replace Nike's logo with a headshot of @thomastaylor312 ;)

**Comment by unguiculus** [comment]
I'm not sure if it makes sense to add logos for individual contributers' companies at all. In any case, I would have to check for codecentric first before we add the logo. Similar to @thomastaylor312 and even though I can do some stuff related to Helm or charts or whatever else during work, I don't officially work on that as codecentric employee.

**Comment by squillace** [comment]
I'd love to add those logos for which we have proud approval to put there. but only with approval. Many people contribute formally, and that's fine. But for those who don't, heck, dunno. Certainly no logos without approval. my two cents, anyway.

**Comment by mgoodness** [comment]
I'm running this up the flagpole at TM, hoping for approval.

**Comment by rji** [comment]
Disclaimer: I am not a lawyer, nor do I represent NIKE, Inc., but I am employed by them.

According to the guidelines at https://help-en-us.nike.com/app/answer/a_id/1071, I do not believe the Nike logo can be used in this context. Further, I would suspect that other organizations have similar logo and trademark restrictions and policies. To echo @squillace's comments, and to avoid any legal ramifications — I would avoid using logos or trademarks without explicit approval.

**Comment by squillace** [comment]
What I'd like to end up with is an ongoing list of contributors, and where desired, those entities that support their work, which would include good ol' capitalist mega-entities as well as non-profits and humans doing well by doing good.

**Comment by mattfarina** [comment]
I am checking with Samsung. There is an approval request process. For now, don't put it up. I'll let you know if/when there is any kind of approval. Being that this is my first time asking about that kind of thing I have no idea how it's going to go.

**Comment by bacongobbler** [comment]
Internally we decided we’re going to remove all logos we don’t have approval at the moment, but we can always add more to the logo wall as @squillace pointed out. That’ll unblock this PR and do what we originally sent out to do: reflect that Helm is a community project; not a Microsoft project. :)

**Comment by unguiculus** [comment]
codecentric's logo can be added.
https://github.com/flynnduism/helm-www/pull/1

**Comment by mgoodness** [comment]
We're approved to add the TM logo. Updated it in the above PR to @flynnduism's fork/branch.

**Comment by flynnduism** [comment]
Thanks for the feedback and logo updates everyone!

I've updated the logo wall and removed any company logos mentioned above that haven't granted permission, and added in a shoutout at the bottom that links to the [contributor list](https://github.com/kubernetes/helm/blob/master/OWNERS).

--

screen shot 2017-11-20 at 9 55 17 am


--

Will do periodic checks to see if anyone wants to add/remove a logo going forward.

**Comment by bacongobbler** [comment]
:shipit:

**Pull request merged**